plugins/woocommerce-blocks/tests/e2e/tests/templates/checkout-template.block_theme.spec.ts

```diff
@@ -29,15 +29,14 @@ test.describe( 'Test the checkout template', () => {
 		editor,
 		page,
 	} ) => {
-		await admin.visitSiteEditor( {
-			postId: templatePath,
-			postType: templateType,
-		} );
-		await admin.visitSiteEditor( { path: '/page' } );
+		await admin.visitSiteEditor( { postType: 'page' } );
 		await editor.page
 			.getByRole( 'button', { name: 'Checkout', exact: true } )
 			.click();
-		await editor.enterEditMode();
+		await page
+			.frameLocator( 'iframe[name="editor-canvas"]' )
+			.locator( 'body' )
+			.click();
```

Comment:
```suggestion
		await editor.canvas.locator( 'body' ).click()
```

Reply:
Updated in e15c594a21ff149034c2e322de62ceb652868782.